webpack.config.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path');
```

Comment:
Unless you are going to get froggy with advanced content, you shouldn't need webpack for your proxy.

Reply:
Yes, I realize that now. I'll nix the file for the time being.